### ISSUE TITLE
Feature request: allow disabling robots.txt and sitemap.xml

### ISSUE THREAD

**Issue by ghost** [Keats/gutenberg#2248]
For a project I'm working I don't need a sitemap.xml nor robots.txt. However there is no way to stop zola generating these files. From what I gather from the docs these can only be overwritten, not disabled entirely. The ignored_content glob also does not seem to have an effect. It would be good if a user had an option to disable these files if they're not wanted.

**Comment by Keats** [comment]
Yeah it's not possible to disable them right now. We could make it a config.toml option. For now you can just rm them after the zola build

**Comment by ghost** [comment]
> For now you can just rm them after the zola build

Thanks, yeah I'll do that for now. Not a huge deal of course. But if it's possible to add as a config value without adding too much complexity I'd appreciate it.

**Comment by Bromind** [comment]
I think we can close this issue now that #2579 is merged.

(ping @Keats ?)

**Comment by Keats** [comment]
I close issues when the feature is released in a Zola version, so it will be closed whenever 0.19.2 or 0.20.0 is released.
